models/public/brain-tumor-segmentation-0002/brain-tumor-segmentation-0002.md

```diff
@@ -13,37 +13,48 @@ This model was created for participation in the [Brain Tumor Segmentation Challe
 | Type              | Segmentation  |
 | GFLOPs            | 300.801       |
 | MParams           | 4.51          |
-| Source framework  | PyTorch       |
+| Source framework  | PyTorch\*       |
 
 ## Accuracy
 
-See [BRATS 2019 Leaderboard](https://www.cbica.upenn.edu/BraTS19/lboardValidation.html). The metrics for challenge validation (Dice_WT, Dice_TC, Dice_ET) differ from the metrics reported below (which are compartible with input labels):
-WT ("whole tumor") class combines all three tumor classes:"necrotic core / non-enhancing tumor", "edema", "enhancing tumor". TC ("tumor core") combines "necrotic core / non-enhancing tumor" and "enhancing tumor".
-
-The following accuracy metrics are measured on a `brain tumor` training subset of [Medical Decathlon](http://medicaldecathlon.com/) dataset.
-Mean:
-Dice index for "overall": 0.915
-Dice index for "necrotic core / non-enhancing tumor": 0.806
-Dice index for "edema": 0.611
-Dice index for "enhancing tumor": 0.794
-Median:
-Dice index for "overall": 0.927
-Dice index for "necrotic core / non-enhancing tumor": 0.835
-Dice index for "edema": 0.644
-Dice index for "enhancing tumor": 0.86
-
-Please, note, that the accuracy achieved with ONNX model adapted for OpenVINO&trade can slightly differ from the accuracy achieved with the original PyTorch model since the upsampling operation has been changed from the `trilinear` to `nearest` mode.
+See [BRATS 2019 Leaderboard](https://www.cbica.upenn.edu/BraTS19/lboardValidation.html). The metrics 
+for challenge validation (Dice_WT, Dice_TC, Dice_ET) differ from the metrics reported below (which 
+are compartible with input labels):
+- WT (*whole tumor*) class combines all three tumor classes:
+    - necrotic core / non-enhancing tumor
+    - edema 
+    - enhancing tumor 
+- TC (*tumor core*) combines the following classes:
+    - necrotic core / non-enhancing tumor
+    - enhancing tumor
```

Comment:
TC stands for necrotic core and non-enhancing tumor, and ET (or DICE_ET) - for enhancing tumor. So correct structure will be something like this:

>- WT (*whole tumor*) class combines all three tumor classes:
>    - necrotic core / non-enhancing tumor
>    - edema 
>    - enhancing tumor 
>- TC (*tumor core*) combines the following classes:
>    - necrotic core 
>    - non-enhancing tumor
> - ET - enhancing tumor

Reply:
Fixed